dotCMS/src/main/java/com/dotcms/rest/api/v1/JobQueueManagerHelper.java

```diff
@@ -0,0 +1,82 @@
+package com.dotcms.rest.api.v1;
+
+import com.dotcms.jobs.business.api.JobProcessorScanner;
+import com.dotcms.jobs.business.api.JobQueueManagerAPI;
+import com.dotcms.jobs.business.processor.JobProcessor;
+import com.dotcms.jobs.business.processor.Queue;
+import com.dotmarketing.util.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.lang.reflect.Constructor;
+import java.util.List;
+
+@ApplicationScoped
+public class JobQueueManagerHelper {
+
+    private JobProcessorScanner scanner;
+
+    @Inject
+    public JobQueueManagerHelper(JobProcessorScanner scanner) {
+        this.scanner = scanner;
+    }
+
+    public JobQueueManagerHelper() {
+    }
+
+    public void registerProcessors(JobQueueManagerAPI jobQueueManagerAPI) {
```

Comment:
wondering why an api is being pass by parameter, would make sense to have it as a property class